admin/gulpfile.js

```diff
@@ -72,9 +72,9 @@ gulp.task('build', ['sass', 'bundleJs', 'copyImages', 'copyPDFs', 'copyCsvFiles'
 
 gulp.task('sass', function () {
   return gulp.src('./assets/**/*.scss')
-    .pipe(sassVariables({
+/*     .pipe(sassVariables({
```

Comment:
why not remove it completely for now? can always revert it through git

Reply:
yes - please remove if it's not in use.

Reply:
@GeorgeChatzigiannis @jon-shipley yep, it should be gone.  commented temporarily to see if all worked ok without it.